### PR TITLE
Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by Sanja969**
**Feature 0:**

- Made content for the readme file
- Made **Person** class with id, name, and age properties in file person
- Made name and get properties to be readable and editable 
- Made id property to be readable
- Made private method _is_of_age? in Person class for checking if age is bigger than 18, and public method which is returning true if _is_of_age? is true
- Made **Student** class that inherits from **Person** class, but it has one more property - classroom
- Made method lay_hooky in Student class that returns '¯\(ツ)/¯'
- Made **Teacher** class that inherits from **Person** class, but it has one more property - specialization
- Overide the method can_use_services? to return always true

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+Feel free to check the [issues page](https://github.com/Sanja969/Decode-a-Morse-code-message/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (muneebulrehman) [inline README.md]: You did a great job adding the README file, however, the title is wrong, the description is missing and also some other sections like setup and prerequisites are missing. Kindly add these sections. Remember it is super important to have a good Readme file.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Sanja969 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It is always better to comment frequently as it becomes easier to debug between small chunks of code. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Sanja969, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Sanja969) [reply README.md]: @muneebulrehman 

- Thank you for noticing that I made mistake in the readme file, I didn't change the title when I used the readme from my last project. I added instructions for installing.

- I think in this case would be weird if I committed more, I just made a few lines of code. Committing more often is better, but in this case really?

**Pull request merged**